diff --git a/node_modules/@polytope-labs/solidity-merkle-trees/src/MerklePatricia.sol b/node_modules/@polytope-labs/solidity-merkle-trees/src/MerklePatricia.sol
index d939b70..945fe28 100644
--- a/node_modules/@polytope-labs/solidity-merkle-trees/src/MerklePatricia.sol
+++ b/node_modules/@polytope-labs/solidity-merkle-trees/src/MerklePatricia.sol
@@ -37,7 +37,7 @@ library MerklePatricia {
      * @return bytes[] a list of values corresponding to the supplied keys.
      */
     function VerifySubstrateProof(bytes32 root, bytes[] memory proof, bytes[] memory keys)
-        public
+        internal
         pure
         returns (StorageValue[] memory)
     {
@@ -106,7 +106,7 @@ library MerklePatricia {
      * @return bytes[], a list of values corresponding to the supplied keys.
      */
     function ReadChildProofCheck(bytes32 root, bytes[] memory proof, bytes[] memory keys, bytes memory childInfo)
-        public
+        internal
         pure
         returns (StorageValue[] memory)
     {
@@ -131,7 +131,7 @@ library MerklePatricia {
      * @return bytes[] a list of values corresponding to the supplied keys.
      */
     function VerifyEthereumProof(bytes32 root, bytes[] memory proof, bytes[] memory keys)
-        public
+        internal
         pure
         returns (StorageValue[] memory)
     {
diff --git a/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol b/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol
index 40b7397..69bee3f 100644
--- a/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol
+++ b/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol
@@ -5,7 +5,6 @@ import "../Bytes.sol";
 import {NibbleSliceOps} from "../NibbleSlice.sol";
 
 import {ScaleCodec} from "./ScaleCodec.sol";
-import "openzeppelin/utils/Strings.sol";
 
 // SPDX-License-Identifier: Apache2
 
