diff --git a/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol b/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol
index 40b7397..69bee3f 100644
--- a/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol
+++ b/node_modules/@polytope-labs/solidity-merkle-trees/src/trie/substrate/SubstrateTrieDB.sol
@@ -5,7 +5,6 @@ import "../Bytes.sol";
 import {NibbleSliceOps} from "../NibbleSlice.sol";
 
 import {ScaleCodec} from "./ScaleCodec.sol";
-import "openzeppelin/utils/Strings.sol";
 
 // SPDX-License-Identifier: Apache2
 
